src/lib/roslibUtils/createSubscriber.ts

```diff
@@ -77,7 +93,7 @@ export default function createSubscriber<T extends TopicType, CB extends boolean
         callback(message);
       }
       if (isDebugging) {
-        console.log(`[${topicName}] Received:`, msg.value);
+        console.log(`[${topicName}] Received:`, msg.value?.data);
```

Comment:
I feel like TypeScript should complain if that's the case. I'll have to take a look at this more closely - I remember I added this because it was just printing out the proxy for the ref.

Reply:
Hm I think we might be talking about different things, but I'll test it out later. The proxy shouldn't be printed out as either case we are accessing the .value. The .data is about like heartbeat which is {data : true} i.e.